docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,368 @@
+
+## Mainnet addresses
+
+```python
+lidoLocator = "0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb"
+withdrawalsVault = 0xB9D7934878B5FB9610B3fE8A5e441e8fad7E293f
+# withdrawalsCredentials = 0x010000000000000000000000b9d7934878b5fb9610b3fe8a5e441e8fad7e293f
+```
+
+## NodeOperatorsRegistry
+
+```python
+stuckPenaltyDelay = 432000  # 5 days as seconds
+```
+
+## OracleDaemonConfig
+
+```python
+NORMALIZED_CL_REWARD_PER_EPOCH=64
+NORMALIZED_CL_REWARD_MISTAKE_RATE_BP=1000
+REBASE_CHECK_NEAREST_EPOCH_DISTANCE=1
+REBASE_CHECK_DISTANT_EPOCH_DISTANCE=23  # 10% of AO 225 epochs frame
+VALIDATOR_DELAYED_TIMEOUT_IN_SLOTS=7200 # 1 day
+VALIDATOR_DELINQUENT_TIMEOUT_IN_SLOTS=28800 # 4 days
+PREDICTION_DURATION_IN_SLOTS=50400
+FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT=1350  # 6 days
+NODE_OPERATOR_NETWORK_PENETRATION_THRESHOLD_BP=100 # 1% network penetration for a single NO
+```
+
+## OracleReportSanityChecker
+
+```python
+churnValidatorsPerDayLimit = 40000
+oneOffCLBalanceDecreaseBPLimit = 500
+annualBalanceIncreaseBPLimit = 1000
+simulatedShareRateDeviationBPLimit = 50
+maxValidatorExitRequestsPerReport = 500
+maxAccountingExtraDataListItemsCount = 500
+maxNodeOperatorsPerExtraDataItemCount = 100
+requestTimestampMargin = 7680
+maxPositiveTokenRebase = 750000
+```
+
+## Burner
+
+```python
+# proofs:
+# - https://vote.lido.fi/vote/106
+# - https://etherscan.io/address/0xB280E33812c0B09353180e92e27b8AD399B07f26#readContract#F7
+totalNonCoverSharesBurnt = 32145684728326685744
+
+totalCoverSharesBurnt = 0  
+_admin = 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c  
+_treasury = 0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c  
+_stETH = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84
+```
+
+## DepositSecurityModule
+
+```python
+# Same as at present https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F10
+maxDepositsPerBlock = 150
+# Same as at present https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F11
+minDepositBlockDistance = 25
+# Same as at present https://etherscan.io/address/0x710B3303fB508a84F10793c1106e32bE873C24cd#readContract#F14
+pauseIntentValidityPeriodBlocks = 6646
+```
+
+## AccountingOracle
+And its corresponding `HashConsensus`.
+
+```python
+epochsPerFrame = 225  # once per day
+# So, the AccountingOracle expected report time would be 12:00 UTC
+
+fastLaneLengthSlots = 10
+lidoLocator = 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb
+lido = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84
+legacyOracle = 0x442af784A788A5bd6F42A01Ebe9F287a871243fb
+```
+
+## ValidatorsExitBusOracle
+And its corresponding `HashConsensus`.
+
+```python
+epochsPerFrame = 75  # thrice per day
+# So, the ValidatorsExitBusOracle expected report time would be 
+# 4:00 UTC, 12:00 UTC, 20:00 UTC
+
+fastLaneLengthSlots = 10
+lidoLocator = 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb
+```
+
+## AccountingOracle and ValidatorsExitBusOracle
+
+```python
+secondsPerSlot = 12
+genesisTime = 1606824023
```

Comment:
https://blog.ethereum.org/2020/11/27/eth2-quick-update-no-21
> This week, the [deposit contract](https://etherscan.io/address/0x00000000219ab540356cbb839cbe05303d7705fa) hit the minimum threshold of 16384 32-ETH validator deposits, kicking genesis into motion. Genesis time is 1606824023 -- that is, December 1, 2020, 12:00:23pm UTC.

Fun fact: mainnet config has only `MIN_GENESIS_TIME`
https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L28

Reply:
should be get from CL node `/eth/v1/beacon/genesis`